src/actions/sources/loadSourceText.js

```diff
@@ -103,5 +108,9 @@ export function loadSourceText(source: SourceRecord) {
     // signal that the action is finished
     deferred.resolve();
     requests.delete(id);
+
+    const TELEMETRY_END = performance.now();
+    const duration = TELEMETRY_END - TELEMETRY_START;
```

Comment:
minor nit: not sure about the coding conventions in the project, but all uppercase for telemetry start/end feels a bit weird. I would expect all uppercase to be file constants.